package.json

```diff
@@ -27,77 +27,77 @@
     "watch": "concurrently npm:watch:*"
   },
   "devDependencies": {
```

Comment:
Shall we take this opportunity to bump dependencies that Github detects as security vulnerabilities: 
I don't have the view in `/arcs`, but this is the view in `/arcs-live`: https://github.com/PolymerLabs/arcs-live/network/alerts

Reply:
The thought was to freeze the versions as-is and only bump after a release.

I'd hold off on security vulnerability reported updates unless there was a valid threat model and some sense of seriousness.

Reply:
Sorry I merged before I saw this. In any case, I think we should be stable for a short period before making any more changes.  Also, I think it's important we try out changes on as many systems as we can before we change committed `package.json`.